rpc/src/v1/types/ibc.rs

```diff
@@ -13,3 +13,62 @@
 //
 // You should have received a copy of the GNU Affero General Public License
 // along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+use codechain_core::ibc::client_02::types::{ClientState as CoreClientState, ConsensusState as CoreConsensusState};
+use primitives::{Bytes, H256};
+
+/// Many of RPC responses will be expressed with this
+/// Because of the nature of IBC, they commonly
+/// 1. Requires a block number for which proof stands
+/// 2. The data should be transparent:
+/// relayer must be able to open it and extract required infomation
+/// 3. Includes a cryptographical proof of that
+#[derive(Debug, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct IBCQuery {
+    pub number: u64,
+    pub data: String,
+    pub proof: Bytes,
+}
+
+#[derive(Debug, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct IBCQueryAbsence {
+    pub number: u64,
+    pub proof: Bytes,
+}
```

Comment:
When we will use `IBCQueryAbsence` type?
IMO, we should make `IBCQuery.data` as optionable.